src/_assets/js/just-the-docs.js

```diff
@@ -65,9 +65,9 @@ function links() {
 			.replace('{link}', item.link)
 			.replace('{name}', item.name)
 			.replace('{description}', item.description)
-			.replace('{img}', item.img);
+			.replace('{img}', item.img || '');
 
-		if (!item.img) {
+		if (item.img == null) {
```

Comment:
Out of curiosity: isn't `!item.img` the same here as `item.img == null` since the simple equality comparison would treat null, false, empty strings, etc all the same?

Reply:
Sure. I think I fixed it by using the `|| ''` further up. I don't know. Doesn't throw the error anymore :)